### DISCUSSION TITLE
【求助】绳匠救救，俺想知道考研数学该怎么复习（＞人＜；）

### DISCUSSION THREAD

**Discussion by tyhpoi** [share121/inter-knot#794] [其他]
![胡桃图](https://github.com/user-attachments/assets/c24b5980-edbe-404f-98d6-b8fc4061b725)
俺是一名准大三生，但距离暑假结束只剩十几天了，可我对考研数学怎么复习仍然很迷茫，不清楚该看谁的视频，该买啥样的书，帮帮我聪明的绳匠先生o((>ω< ))o

**Comment by tyhpoi** [comment]
有人接委托吗？迷茫大雪牲在线求助(。>︿<)_θ

**Reply by JiangHuai-H** [reply]
你好（自动回复）

**Comment by WLMRMZ** [comment]
这个委托我本来是想接的，可是我不会，没有关系，我会转告其他绳匠的<(￣︶￣)↗[GO!]

**Comment by cloud-oc** [comment]
是下学期大三吗，是的话就和我一样，可以用《考研数学复习全书》，不过我的还在吃灰^_^
